From c84addba46bb7e9caea0971b137501de2b94741f Mon Sep 17 00:00:00 2001
From: linmin <linmin@eswincomputing.com>
Date: Wed, 12 Jun 2024 09:35:13 +0800
Subject: [PATCH 085/109] WIN2030-15279:chore(hf_common):BMC version update to
 2.0

Changelogs:
1.BMC software version update to 2.0, the major version 2 stands
  for DVB-2

Change-Id: I1ab64d434f32f7ba35b96ae80a871f3aaffbb7dc
---
 Core/Inc/hf_common.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Core/Inc/hf_common.h b/Core/Inc/hf_common.h
index 0a19b90..de1be56 100644
--- a/Core/Inc/hf_common.h
+++ b/Core/Inc/hf_common.h
@@ -215,8 +215,8 @@ extern UART_HandleTypeDef huart3;
 #define SOM_MAC1_IDX	1
 #define MCU_MAC_IDX	2
 /* define ------------------------------------------------------------*/
-#define BMC_SOFTWARE_VERSION_MAJOR                   1
-#define BMC_SOFTWARE_VERSION_MINOR                   2
+#define BMC_SOFTWARE_VERSION_MAJOR                   2
+#define BMC_SOFTWARE_VERSION_MINOR                   0
 
 #define MAGIC_NUMBER	0x45505EF1
 
-- 
2.25.1

